kv/kvserver/loqrecovery: TestRejectBadVersionApplication failed

kv/kvserver/loqrecovery.TestRejectBadVersionApplication [failed](https://mesolite.cluster.engflow.com/invocations/default/335db883-6df0-4137-864b-0a1a190462a5?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZzZXJ2ZXIvbG9xcmVjb3Zlcnk6bG9xcmVjb3ZlcnlfdGVzdA==) on master @ [d87d47386f47e826d4694fa50113a1d9ff6a9612](https://github.com/cockroachdb/cockroach/commits/d87d47386f47e826d4694fa50113a1d9ff6a9612):

Fatal error:

```
panic: test timed out after 5m0s
running tests:
	TestRejectBadVersionApplication (3s)

```
Stack: 

```
goroutine 291462 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x385
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRejectBadVersionApplication
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestRejectBadVersionApplication2932405552
    test_log_scope.go:76: use -show-logs to present logs inline
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRejectBadVersionApplication.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver/loqrecovery.TestRejectBadVersionApplication [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18030065?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18030065?buildTab=artifacts#/) on master @ [78e363f4cbd720292aad2a3bed24790cd1573fd2](https://github.com/cockroachdb/cockroach/commits/78e363f4cbd720292aad2a3bed24790cd1573fd2):

Fatal error:

```
panic: test timed out after 5m0s
running tests:
	TestRejectBadVersionApplication (6s)

```
Stack: 

```
goroutine 327569 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRejectBadVersionApplication
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/ea0927e04eade2740899effbae326bb2/logTestRejectBadVersionApplication2131376456
    test_log_scope.go:76: use -show-logs to present logs inline
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRejectBadVersionApplication.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/kv/kvserver/loqrecovery/BUILD.bazel
+++ b/pkg/kv/kvserver/loqrecovery/BUILD.bazel
@@ -58,6 +58,7 @@ go_library(
 
 go_test(
     name = "loqrecovery_test",
+    size = "large",
     srcs = [
         "apply_test.go",
         "collect_raft_log_test.go",

